odxtools/cli/find.py

```diff
@@ -0,0 +1,170 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2021 MBition GmbH
+
+import argparse
+import re
+import markdownify
+
+from ..database import Database
+from ..service import DiagService
+from . import _parser_utils
+
+
+def format_desc(desc, ident=0):
+    # Collapse whitespaces
+    desc = re.sub(r'\s+', ' ', desc)
+    # Covert XHTML to Markdown
+    desc = markdownify.markdownify(desc)
+    # Collapse blank lines
+    desc = re.sub(r'(\n\s*)+\n+', '\n', desc).strip()
+
+    if '\n' in desc:
+        desc = '\n' + ident * ' ' + \
+            ('\n' + ident * ' ').join(desc.split('\n'))
+    return desc
+
+
+def print_diagnostic_service(service: DiagService, print_params=False):
```

Comment:
hm, I guess we should probably move the "print service/parameter/etc" functions to a separate file and use them from all tools which need them (`list` and `browse` come to mind). Anyway, that's for another day. (at least if you don't want to jump on this immediately ;))

Reply:
I moved the duplicated functions from list and find into _print_utils.py as a start